cors: add a flag for cors (#5671)
cosmos/cosmos-sdk: docs/interfaces/rest.md

```go
rootCmd.AddCommand(rest.ServeCommand(cdc, registerRoutes))
```

> ## Cors

Reviewer: ```suggestion
## Cross-Origin Resource Sharing (CORS)
```

> Cors are not enabled by default to help with security. If you would like to use the rest-server in a public environment we recommend you provide a reverse proxy, this can be done with [nginx](https://www.nginx.com/). For testing and development purposes there is an `unsafe_cors` flag that can be passed to the cmd to enable accepting cors from everyone.

Reviewer: ```suggestion
CORS policies are not enabled by default to help with security. If you would like to use the REST server in a public environment it is recommended to run the service behind a reverse proxy such as [nginx](https://www.nginx.com/). For testing and development purposes there is an `--unsafe-cors` flag that can be passed to the cmd to enable accepting CORS request from all origins.
```

```sh
gaiacli rest-server --chain-id=test \
    --laddr=tcp://localhost:1317 \
    --node tcp://localhost:26657 \

>     --trust-node=true --unsafe_cors

Reviewer: ```suggestion
    --trust-node=true --unsafe-cors
```
